### PR TITLE
Coalition of Snackolonies II: Fetch Harder Edition

### PR DESCRIPTION
**Pull request by crunchypretzels**
The ghola of my previous PR, which was killed by a series of merge conflicts and programming errors.

This PR adds a number of new lore-related foods from the Coalition.

-Three new alcoholic drinks, two from Xanu Prime and one from Gadpathur, as well as a new cocktail and some canned beer brands.
-Several new non-alcoholic drinks, such as a Galatean 'energy drink', several brands of Xanan refreshment, mushroom kvass, and some Vysokan... ichor soda.
-Several new cooking recipes for Xanu Prime and Himeo, as well as the much-awaited chicken parm hero.
-More than a dozen new snack foods from across the Coalition of Colonies, as well as a vending machine that holds them all in it. This will be mapped onto the Horizon in the near future, as well as related awaysites.
-Two new mugs; Galatea and the All-Xanu Republic, which can be found in the loadout.
-Hakhma meat; butcher a hakhma.

All lore comes from the Human Lore Team. The sprites are by me, or are modified versions of sprites provided by our esteemed playerbase.

A thousand thanks to the half-dozen people who put this together, especially Lain, who came up with all the Xanan food and drinks, Schwann, who came up with the Himean and Galatean contributions, and Wiggles, who came up with all the Vysokan content.

image

### CODE REVIEW

**Comment by crunchypretzels** [comment]
!review

**Review by DreamySkrell — APPROVED** [review]

File: code/game/machinery/vending_types.dm [hunk 1]
```diff
@@ -1644,6 +1648,90 @@
...
 
+/obj/machinery/vending/frontiervend
+	name = "\improper FrontierVend"
+	desc = "A vending machine specialized in snacks from the Coalition of Colonies."
+	desc_extended = "Almost rebranded to the 'Coalition of Snackolonies', the FrontierVend brand is owned by a now-subsidiary of Orion Express specialized in food exports. These machines \
+	are omnipresent throughout the settled regions of human space, forming a sort of Coalition superculture; it's easier to sympathize with someone if you eat the same snacks."
+	icon_state = "frontiervend"
+	icon_vend = "frontiervend-vend"
+	//product_slogans = "At least 85 billion served!;A new frontier of flavors!;Snacking for a free frontier!;WARNING: This machine does not accept Solarian credits."
+	//product_ads = "Kill Solarians.;Behead Solarians.;Roundhouse kick a Solarian into the concrete.;Slam-dunk Solarians into the trashcan.;Launch Solarians into the sun."
```
Comment (NonQueueingMatt) [inline code/game/machinery/vending_types.dm:1659]: remove this commented code if it's not implemented

File: code/game/machinery/vending_types.dm [hunk 2]
```diff
@@ -1644,6 +1648,90 @@
...
+		/obj/item/reagent_containers/food/snacks/chana = 18,
+		/obj/item/reagent_containers/food/snacks/chana/wild = 18,
+		/obj/item/reagent_containers/food/snacks/papad = 15,
+		/obj/item/reagent_containers/food/snacks/papad/garlic = 15,
+		/obj/item/reagent_containers/food/snacks/papad/ginger = 15,
+		/obj/item/reagent_containers/food/snacks/papad/apple = 15,
+		/obj/item/storage/box/fancy/foysnack = 25
+	)
+	contraband = list(
+	//	/obj/item/storage/box/fancy/yoke/earthmover = 2 // guys rocked a little too hard for the GC but we will never forget the cold ones we cracked
```
Comment (NonQueueingMatt) [inline code/game/machinery/vending_types.dm:1719]: remove commented code if not implemented

File: code/game/machinery/vending_types.dm [hunk 3]
```diff
@@ -1644,6 +1648,90 @@
...
+	)
+	random_itemcount = 0
+	light_color = COLOR_BABY_BLUE
+
+/obj/machinery/vending/frontiervend/hacked
+	name = "hacked FrontierVend"
+	desc = "A complimentary FrontierVend machine. No money? No worries."
+	prices = list()
+
+
```
Comment (NonQueueingMatt) [inline code/game/machinery/vending_types.dm:1734]: extra space

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**